README.md

```diff
@@ -132,7 +132,7 @@ This repository contains many companion libraries to write integration tests wit
 | Helpful output to find slow tests                                                         | ✔️       | ✔️                                                           | C                                                                |
 | Coloured output                                                                           | ✔️       | ✔️                                                           | ✔️                                                                |
 | Colourless output                                                                         | ✔️       | ✔️                                                           | ✔️                                                                |
-| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
+| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
```

Comment:
Hspec does not use `ppShow`, but it provides a `--pretty` option which is similar in spirit (enabled by default) (since 2.9.3).

NB: The pretty printing is configurable through `configPrettyPrintFunction`, so if you are inclined, you could also use `ppShow`.

Reply:
Then let's make that feature "Pretty printing counterexample output" because I'm not married to `ppShow` either.

Reply:
That's OK with me, but then I think maybe at at least apply it uniformly, specifically:

---
![IMG_1661](https://github.com/user-attachments/assets/f22fe3e1-00b7-48c4-a20e-3966eda511fe)
---

Reply:
Done.